.changeset/itchy-files-jam.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/linear-finance-adapter': minor
```

Comment:
We make a pretty significant change from reading local allocations to requiring an API key for an external call. Do you think it makes sense to make this a major bump instead?

Reply:
hmm fair point.  Let's make it major.